.github/workflows/build_windows_packages.yml

```diff
@@ -67,6 +67,10 @@ jobs:
           choco install --no-progress -y awscli
           echo "$PATH;C:\Program Files\Amazon\AWSCLIV2" >> $GITHUB_PATH
 
+      - uses: iterative/setup-dvc@4bdfd2b0f6f1ad7e08afadb03b1a895c352a5239 # v2.0.0
+        with:
+          version: '3.62.0'
```

Comment:
dvc will also need to be installed in these workflows:

* https://github.com/ROCm/TheRock/blob/main/.github/workflows/release_windows_packages.yml
* https://github.com/ROCm/rocm-libraries/blob/develop/.github/workflows/therock-ci-windows.yml
* https://github.com/ROCm/rocm-systems/blob/develop/.github/workflows/therock-ci-windows.yml

If we include it in the base VM image that all of our build machines use that will be simpler, or we could add a common "install_ci_requirements.py" script that runs pip installs, choco installs, etc. If we wanted that "script" to run other github actions like this one, it would need to be a reusable action... which I'm not _totally_ opposed to, but I'd rather avoid it.